src/plugins/intel_gpu/src/graph/impls/cm/xetla_lstm_seq.hpp

```diff
@@ -45,6 +45,7 @@ struct LSTMSeqImplementationManager : public ImplementationManager {
         const auto& info = engine.get_device_info();
 
         // XeTLA LSTM optimized for Xe2 architectures
+        std::cout << "check_cm_jit_support called at " << __func__ << ", L" << __LINE__ << std::endl;
```

Comment:
Debug message? Could be removed?

Reply:
@praasz Will remove them later, they are being used to debug an issue on CI machine only.